semgrep-core/src/analyzing/Dataflow_tainting.ml

```diff
@@ -77,6 +78,49 @@ let option_to_varmap = function
   | Some lvar -> VarMap.singleton (str_of_name lvar) ()
 (*e: function [[Dataflow_tainting.option_to_varmap]] *)
 
+(*****************************************************************************)
+(* Tainted *)
+(*****************************************************************************)
+
+let sanitized config instr =
+  match instr.i with
+  | Call (_, {e=Lvalue({base=Var(("sanitize",_),_);_}); _}, [])
```

Comment:
I would add -> true for the Call case and
then just do | _ -> config.is_sanitizer instr

Reply:
Good point, fixed.